backend/docs/GITHUB-ACTIONS.md

```diff
@@ -0,0 +1,59 @@
+# GitHub Actions
+
+GitHub Actions provide a CI/CD workflow abilities configured through YAML that will allow us to build and test our code before a Pull Request is created and merged.
```

Comment:
/nit: the actions run after a PR is created but before they get merged

Reply:
I will have to look into seeing if we can pray to rocket chat on a failure at some point